document/chapters/03-sdf_preconditioner.tex

```diff
@@ -1183,6 +1183,106 @@ \subsection{Memory Usage}
+error in the particle tracking algorithm. In this scenario, particles can
+numerically leave the volume without updating the material in which the particle
+is interacting in the physics kernel, resulting in inaccurate physics
+events. This will continue until the particle is killed due to its low
+statistical contribution to the simulation solution. Occasional
+\textit{point\_containment} checks in the particle tracking algorithm can detect
+these particles and label them as lost, but this is not guaranteed to happen and
+the simulation may run without reporting any particle tracking errors. Creating
+a fully sealed DAGMC model from a general CAD representation is a difficult task
+for many complex production models, requiring that the model is created in a
+friendly way such that all volumes can be imprinted and merged successfully
+without overlaps. Corner cases also exist in which the \textit{make\_watertight}
+algorithm fails to fully seal tessellations of the CAD model \cite{Smith_2011}.
+
+In fully sealed models, however, this optimization is robust and can
+significantly decrease simulation times without consequence. This optimization
+has been applied to the same set of production models in Table
+\ref{tab:sdf_prod_summary} to compare the improvement in performance seen when
+using this mode in comparison to the similar implicit tracking method employed
+using the SDF preconditioner.
+
+It is worth nothing that the errant particle pathology associated with the
+physics distance limit is not present when using SDF preconditioning to
+accelerate DAGMC simulations. SDF values are generated using known surface
+locations relative to points in the uniform mesh. As such, information gathered
+from the SDF during preconditioning of particle queries will not allow particles
+to cross surfaces so long as the model is well-formed. This pathology is avoided
+by accounting for the interpolation error associated with signed distance value
+calculation to avoid incorrectly preconditioned ray queries. Rays fired in
+simulations where SDF preconditioning is applied still have an effectively
+infinite length and thus do not invalidate the robust particle tracking process
+in DAGMC. Particles moving through gaps in unsealed volumes will always be
+reported by DAGMC.
+
+\begin{table}[H]
+  \centering
+  \begin{tabular}{c c c c c}
+    \toprule
+    \textbf{\thead{Model}} & \textbf{\thead{Performance \\ Ratio}} & \textbf{\thead{Fully Sealed \\ Model}} & \textbf{\thead{Matching \\ Results}} & \textbf{\thead{No Lost \\ Particles}} \\
+    \hline
+    ITER & 0.23 & $\times$ & $\times$ & $\times$ \\
+    nTOF & 0.46 & $\times$ & $\checkmark$ & $\checkmark$ \\
+    SHINE  & 0.86 & $\checkmark$ & $\checkmark$ & $\checkmark$ \\
+    SHINE* & 0.51 & $\checkmark$ & $\checkmark$ & $\checkmark$ \\
+    SNS    & 0.95 & $\times$ & $\checkmark$ & $\checkmark$ \\
+    \bottomrule
+  \end{tabular}
+  \caption[SDF production simulation times using DAGMC's physics distance
```

Comment:
This caption is awkward/unclear.  I think the above paragraph suggests that you are comparing SDF-only to DAGMC distance limit only.  This caption makes it seem like you are comparing distance limit + SDF vs SDF-only.  And I'm not sure what's in the numerator/denominator of this ratio.

Reply:
Good point. I'll make the table header more clear and the preceding paragraph more clear.